src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationParameters.java

```diff
@@ -67,6 +71,14 @@ private static final int checkMaxNumberIterations(int maxNumberIterations) {
         return maxNumberIterations;
     }
 
+    public boolean getConstantPowerScaling() {
```

Comment:
Maybe rename it `isConstantPowerScaling` when the getter returns a boolean

Reply:
It should be indeed `isLoadPowerFactorConstant`.